### ISSUE TITLE
Console logging appender toggle

### ISSUE THREAD

**Issue by xmorose** [DiscordSRV/DiscordSRV#1386]
### Why?

Having huge chonky blocks of text getting appended to each other until messages maxing out, and then sending the next huge block is incredibly infuriating and just adds to server owner's spent time on skimming through logs / searching. 
You cannot search properly like this, 2 exampleimages are attached, while one is using the old logging with proper singular messages, and the other one is using the new logging
![Wherethefuck](https://user-images.githubusercontent.com/53750990/161357938-4250b5f7-d4b3-4f67-a86a-836bf10a0862.png)
![therewego](https://user-images.githubusercontent.com/53750990/161357939-27092595-0541-49f9-ad05-a974c13722a5.png)
.

### What and How?

Having a config option called "AppendNewMessages" or something like that would be best, toggling between singular-messages (or at least, accumulate everything that got logged in the console in the past x seconds configured in the "DiscordConsoleChannelLogRefreshRateInSeconds"
And full appends maxing out messages through editing the original message, and then sending a new one if the previous one got maxed out.

### Are there alternatives?

The only way that "works quite well for me" is downgrading the discordsrv version, which isnt really practical but is like the only way i was able to find for this, which is why a config toggle option would be great, for anyone that still prefers the new logging.

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by akdukaan** [comment]
I'm commenting to add support for this enhancement. This new method makes search a lot more difficult with larger blocks while also causing some issues with Discord's search indexing. 

Ever since this was changed, I've had an issue where messages in console aren't searchable. Everything is fine and normal up to a certain point, but then after a few days of using the appending method, it seems Discord stops indexing new messages that appear in that channel. Any message sent after that point, whether its by the DiscordSRV bot or a person, doesn't show up in any search results. I've had to recreate the console channel twice so far so that I could search again.

I've also noticed that a few other people have been having the same issue recently too. 
- https://discord.com/channels/135634590575493120/135634590575493120/999876015042076774
- https://discord.com/channels/135634590575493120/135634590575493120/997871479037370538

I haven't heard anyone have the issue before 1.25.0 when console appending was added. If I had to guess, something about editing the messages uses a lot of Discord's resources and to preserve them, Discord simply stops indexing messages in that channel from the search.

I also know that a lot of other people want it to go back to the way it used to be. 
- https://discord.com/channels/135634590575493120/135634590575493120/988870865183703110
- https://discord.com/channels/135634590575493120/135634590575493120/1001125674968416296
- https://discord.com/channels/135634590575493120/135634590575493120/986830601216745524
- https://discord.com/channels/135634590575493120/135634590575493120/981838553581490226
- https://discord.com/channels/135634590575493120/135634590575493120/993404742879428699

**Comment by GreemDev** [comment]
You can disable this in the config.

**Comment by Vankka** [comment]
You can disable it being made into a code block, yes but you cannot disable it from editing messages instead of sending news ones which is the the topic of this ticket

**Comment by xmorose** [comment]
Is there any update on this?

**Comment by GreemDev** [comment]
If there was, it would be updated.

**Comment by accLarson** [comment]
I'm looking forward to the option to toggle the `consoleAppender`.

Although I'm sure the optimization from the appender is significant, it severely reduces the effectiveness of the console channel for me. As has been mentioned above, searching the console channel with appended message is a hassle. What use to take a minute to scan 30+ search results now requires much longer, perhaps 10x as long since I must scan each result for my actual search term(s).

It's such a huge benefit to to have my console output stored in a channel which can be searched! Within moments I can search any term in console and get results as early as 2018 (when I began using DiscordSRV). Since the appender was added, it's been manageable but a reduced QOL in terms of searchability.

Is this enhancement actively planned?

**Comment by Dinty1** [comment]
Yes, it is planned. Bumping this with the same information as everyone else won’t speed it up 🙂

**Comment by Dinty1** [comment]
This has been implemented on v2 (testing builds available from our discord server)
